Project.toml

```diff
@@ -21,6 +21,7 @@ NLsolve = "2774e3e8-f4cf-5e23-947b-6d7e65073b56"
 NearestNeighbors = "b8a86587-4115-5ab1-83bc-aa920d37bbce"
 QuadGK = "1fd47b50-473d-5c70-9696-f719f8f3bcdc"
 Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
```

Comment:
Remove Revise package from Project.toml. You can add it to the main environment (e.g., v1.6) and you still have access even if the environment loaded into the Package manager is the one of Gridap.

Reply:
done

Reply:
ok, thanks